src/app/pages/card-import/card-import.page.ts

```diff
@@ -19,6 +21,9 @@ export class CardImportPage implements OnInit {
     backImg: ''
```

Comment:
For simplicity, please use the Card interface provided by the Card Service.

Reply:
Yep, these are the changes that need to be made as a result of your changes in the card service.